app/main.js

```diff
@@ -94,8 +94,9 @@ function finishMicrobreak (shouldPlaySound = true) {
   planMicrobreakTimer = setTimeout(planMicrobreak, 100)
 }
 
-function planMicrobreak () {
-  startMicrobreakTimer = setTimeout(startMicrobreak, settings.get('microbreakInterval'))
+function planMicrobreak (time) {
+  let startTime = (typeof time === 'number') ? time :settings.get('microbreakInterval')
```

Comment:
As this is at `app.on('ready', planMicrobreak)`, it's kind of hack to check if it is a number, as it is another object at the time of this call. Maybe we can find better way? 

I am thinking about: 

```
app.on('ready', planFirstBreak)

function planFirstBreak () {
  planMicrobreak()
}

function planMicrobreak (startTime = settings.get('microbreakInterval')) {
  startMicrobreakTimer = setTimeout(startMicrobreak, startTime)
}
```

I find this more readable, what do you think?